sources/lib/api/gcp/bigquery/_schema.py

```diff
@@ -277,3 +277,7 @@ def __eq__(self, other):
       if not self._map[name] == other_map[name]:
         return False
     return True
+
+  def __ne__(self, other):
```

Comment:
Both are implemented. We need to implement both because in each case this is a structural comparison, not referential comparison.